zproject/prod_settings_template.py

```diff
@@ -57,6 +57,17 @@
 #FAKE_EMAIL_DOMAIN = 'fake-domain.example.com'
 
 
+################
+# Public Listing Of Realms.
+#
+# If you need people to be able to list their realms on the
+# communities page, you should set the OSS_SHOWCASE to True.
+#
+# To avoid spam currently only sponsored realms can list their
+# realms publicly.
+OSS_SHOWCASE = False
```

Comment:
We don't want to document this setting here; instead document it in `zproject/default_settings.py`

Since we don't expect it to be useful to self-hosted systems.